packages/aws-cdk/test/cli/cdk-toolkit.test.ts

```diff
@@ -216,7 +216,7 @@ describe('deploy', () => {
       // WHEN
       await cdkToolkit.deploy({
         selector: { patterns: ['Test-Stack-A-Display-Name'] },
-        requireApproval: RequireApproval.Never,
+        requireApproval: RequireApproval.NEVER,
```

Comment:
this is kind of silly but also a breaking change right? what should i have done instead?

Reply:
why is it breaking?

Reply:
i suppose its not because the CLI users have no access to this..